lib/rails_cloud_tasks/scheduler.rb

```diff
@@ -53,7 +45,7 @@ def build_job(job)
         description: job[:description],
         time_zone:   job[:time_zone],
         http_target: {
-          uri:         "#{host}#{tasks_path}/#{job[:name]}",
+          uri:         "#{host}#{tasks_path}/#{job[:class_name] || job[:name]}",
```

Comment:
I dislike this approach of having two distinct parameters to set the same property.
If the task name cannot contain colons, then we shouldn't use it to select the job class.
My suggestion is to make the `:class_name` attribute mandatory and use only it to select the job class.